web/src/containers/admin/FederalAdmin.js

```diff
@@ -1,15 +1,267 @@
-import React from 'react';
+import PropTypes from 'prop-types';
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import { Button, Tabs, TabPanel } from '@cmsgov/design-system';
 
-// Note: This component is a placeholder until we implement it in story #2812
-const FederalAdmin = () => {
+import {
+  getRoleTypes,
+  getAffiliations,
+  updateAffiliation
+} from '../../actions/admin';
 
+import { getUserStateOrTerritory } from '../../reducers/user.selector';
+
+import ManageRoleDialog from './ManageRoleDialog';
+import ConfirmationDialog from './ConfirmationDialog';
+import ManageAllUsersTable from './ManageAllUsersTable';
+
+
+const FederalAdmin = ({
+  currentState,
+  currentUser,
+  currentAffiliations,
+  roleTypes,
+  getRoleTypes: fetchTypes,
+  getAffiliations: affiliations,
+  updateAffiliation: actualUpdateAffiliation
+}) => {
+  const [activeTab, setActiveTab] = useState('pending');
+  const [isFetching, setIsFetching] = useState(true);
+  const [isDenied, setIsDenied] = useState(true);
+
+  const [selectedAffiliation, setSelectedAffiliation] = useState();
```

Comment:
Knoll suggested turning these into an object that contains the affiliation, id, and state. I went that route since were not doing anything beyond setting and getting them. What do you think?